src/components/v5/shared/UserInfoPopover/partials/PermissionTooltip/PermissionTooltip.tsx

```diff
@@ -0,0 +1,77 @@
+import { ColonyRole } from '@colony/colony-js';
+import { User, UsersThree } from '@phosphor-icons/react';
+import React, { type FC } from 'react';
+import { defineMessages } from 'react-intl';
+
+import { type AvailablePermission } from '~hooks/members/types.ts';
+import Tooltip from '~shared/Extensions/Tooltip/Tooltip.tsx';
+import { formatText } from '~utils/intl.ts';
+import PermissionsBadge from '~v5/common/Pills/PermissionsBadge/PermissionsBadge.tsx';
+
+import PermissionTooltipContent from './PermissionTooltipContent.tsx';
+
+const displayName = 'v5.UserInfoPopover.partials.PermissionTooltip';
+
+const MSG = defineMessages({
+  multiSigPrepend: {
+    id: `${displayName}.multiSigPrepend`,
+    defaultMessage: 'Multi-sig ',
+  },
+});
+
+interface PermissionTooltipProps {
+  isMultiSig?: boolean;
+  isRootDomain?: boolean;
+  userPermissionsInDomain: AvailablePermission[];
+  userPermissionsInParentDomain: AvailablePermission[];
+}
+
+const PermissionTooltip: FC<PermissionTooltipProps> = ({
+  isMultiSig = false,
+  isRootDomain = false,
+  userPermissionsInDomain,
+  userPermissionsInParentDomain,
+}) => {
+  const getInheritedPermissions = (): AvailablePermission[] => {
+    if (isRootDomain) {
+      return [];
+    }
+
+    return userPermissionsInParentDomain
+      .filter(
+        (permission) =>
+          permission !== ColonyRole.Root && permission !== ColonyRole.Recovery,
+      )
+      .filter((permission) => !userPermissionsInDomain.includes(permission));
```

Comment:
Nitpick, but we could just combine these two filters into one so that it doesn't loop over the array twice. Could also just have userInheritedPermissions be calculated inside a useMemo so that the function isn't called on each render.